apps/admin/backend/src/util/exporter.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Exporter,
+  UsbDrive as DeprecatedUsbDriveType,
+} from '@votingworks/backend';
+import { UsbDrive } from '@votingworks/usb-drive';
+import { ADMIN_ALLOWED_EXPORT_PATTERNS } from '../globals';
+
+/**
+ * TODO: Convert Exporter to use libs/usb-drive types.
+ *
+ * Wraps the libs/usb-drive interface in a `getUsbDrives` function
+ * which is the pattern currently expected the libs/backend.
+ */
+export function getGetUsbDrives(
```

Comment:
I might name this something like `buildGetUsbDrivesFunction` to make it clearer that it's not typo'd from `getUsbDrives`.